### ISSUE TITLE
FV Indicator support incompatible with sparse AD

### ISSUE THREAD

**Issue by loganharbour** [idaholab/moose#16559]
## Reason
See https://civet.inl.gov/job/650937/. Sparse AD is not working with FV indicators.

FYI @cpgr 

Refs #16081.

## Design
TBD.

## Impact
Compatibility with sparse AD and FV indicators.

**Comment by cpgr** [comment]
I'll take a look at this today - is it just a matter of running `./configure --with-derivative-type=sparse --with-ad-indexing-type=global` to replicate this sparse AD system?

**Comment by loganharbour** [comment]
Correct. After looking at the PR - I suspect the fix may not be too obvious. Ping @rwcarlsen

**Comment by loganharbour** [comment]
Adaptivity is being removed in #16563 to get tests passing again - this is the failing test before the change: https://github.com/idaholab/moose/blob/bd7a9977c06e464d9ef3476e50fcde2ecfb4209c/test/tests/markers/error_fraction_marker/error_fraction_marker_fv.i
